notebooks/3_Create_Update_Insert.ipynb

```diff
@@ -713,7 +660,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "c.sequences.retrieve(id=2357839693666680).columns"
+    "c.sequences.retrieve(external_id=prefix+\"_test_sequence\").columns"
```

Comment:
This looks a bit weird now; why the sudden jump from datapoints to sequence data?

Reply:
will look into it